benchmarks/benchmarks/bench_io.py

```diff
@@ -67,6 +67,17 @@ def time_vb_savez_squares(self):
         np.savez('tmp.npz', **self.squares)
 
 
+class LoadNpyOverhead(Benchmark):
+    # inspired by https://github.com/divideconcept/fastnumpyio
+    
```

Comment:
```suggestion
```
Maybe keep it in the commit message?  I will doubt it adds much in 5 years?  But I don't care much.

Reply:
I was thinking about it, too. The code I've provided diverged over the course of iterations anyway. I'll delete that comment.